xml/obs/openSUSE:Leap:15.3:WSL.xml

```diff
@@ -1,5 +1,5 @@
 <openQA project_pattern="openSUSE:Leap:(?P&lt;version&gt;15.3):WSL"
-    dist_path="obspublish::openqa/openSUSE:Leap:15.3:WSL/standard/x86_64/wsl-appx">
+    dist_path="obspublish::openqa/openSUSE:Leap:15.3:WSL/images">
```

Comment:
It probably should be `images/x86_64/kiwi-images-wsl/` : 
```
> rsync obspublish::openqa/openSUSE:Leap:15.3:WSL/images/*appx
rsync: link_stat "/openSUSE:Leap:15.3:WSL/images/*appx" (in openqa) failed: No such file or directory (2)
rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1674) [Receiver=3.1.3]
> rsync obspublish::openqa/openSUSE:Leap:15.3:WSL/images/x86_64/kiwi-images-wsl/*appx
-rw-r--r--    182,353,539 2021/06/01 13:36:26 openSUSE-Leap-15.3-WSL.x86_64-153.1.1.0-Build1.1.appx
```

Reply:
Updated, thanks!